tsMuxer/aacStreamReader.cpp

```diff
@@ -9,7 +9,7 @@
 #include "vodCoreException.h"
 #include "vod_common.h"
 
-unsigned AACStreamReader::getHeaderLen() { return AAC_HEADER_LEN; }
+int AACStreamReader::getHeaderLen() { return AAC_HEADER_LEN; }
```

Comment:
If this is an actual length, it shouldn't be negative : if anything, it's `AAC_HEADER_LEN`'s type that should be changed to unsigned.

Reply:
If getHeaderLen() is changed to unsigned type, then we would have to cast e.g. `if (m_bufEnd - m_curPos < getHeaderLen())` and other variables.

There is no real reason to use an unsigned type for audio header length `getHeaderLen()`, as the value is guaranteed to be below 14: returned values LPCM = 4, AAC = 7, AC3 = 7, DVB = 10, DTS = 14